### ISSUE TITLE
Using silk touch on broken vault blocks is a bad idea

### ISSUE THREAD

**Issue by CplPibald** [ShetiPhian/Localizations-and-Issues#97]
multistorage-1.11.0-1.3.7b

One of my players discovered that using silk touch on broken vault blocks (maybe all vault blocks) produces an "unused item" with no texture.  When placed, these blocks look like regular completed vault blocks, but we can't create a vault from them.  Trying to place them throws the following error in the server log.

```
[22:59:57] [Server thread/ERROR] [FML/]: A TileEntity type shetiphian.multistorage.common.tileentity.TileEntityVaultBroken has throw an exception trying to writ
e state. It will not persist. Report this to the mod author
java.lang.NullPointerException
        at shetiphian.multistorage.common.tileentity.TileEntityVaultBroken.buildNBT(TileEntityVaultBroken.java:15) ~[TileEntityVaultBroken.class:?]
        at shetiphian.core.common.tileentity.TileEntityBase.func_189515_b(TileEntityBase.java:80) ~[TileEntityBase.class:?]
        at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75820_a(AnvilChunkLoader.java:392) [avd.class:?]
        at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75816_a(AnvilChunkLoader.java:184) [avd.class:?]
        at net.minecraft.world.gen.ChunkProviderServer.func_73242_b(ChunkProviderServer.java:203) [lv.class:?]
        at net.minecraft.world.gen.ChunkProviderServer.func_186027_a(ChunkProviderServer.java:231) [lv.class:?]
        at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1015) [lw.class:?]
        at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:366) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.func_71260_j(MinecraftServer.java:404) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:535) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]

```

Further trying to interact with these blocks crashed my server with no log.  I am still investigating.

**Comment by ShetiPhian** [comment]
Looks like I didn't exclude the broken blocks for the silk touch handler.

**Comment by ShetiPhian** [comment]
Fixed in 1.3.9
https://minecraft.curseforge.com/projects/multistorage/files/2470620
